ios/FluentUI/Core/ColorProviding.swift

```diff
@@ -100,14 +100,13 @@ private func brandColorOverrides(provider: ColorProviding,
     /// - Parameters:
     ///   - provider: The `ColorProvider2` whose colors should be used for controls in this theme.
     @objc(setColorProvider:)
-    func setColorProvider(_ provider: ColorProviding) {
+    @discardableResult
+    func setColorProvider(_ provider: ColorProviding) -> FluentTheme {
```

Comment:
Is this going to be a breaking change? Is that what @discardableResult prevents?

Reply:
It potentially could be - but luckily as @laminesm pointed out, this isn't really necessary for this PR so I'm removing the return value altogether for now.